bin/setup

```diff
@@ -2,12 +2,8 @@
 
 set -e
 
-# Install and initialize govendor
-go get -u github.com/kardianos/govendor
-govendor init
-
-# Update Go dependencies
-govendor sync
+# Get dependencies
+go get -u ./...
```

Comment:
is the `./...` just an alias for the repo url?

Reply:
The `go get` command adds dependencies (which could be "modules" or "packages")
to the current development module and then builds and installs them.

It accepts arguments and its first step is to resolve which dependencies to add.

We could provide a single name as an argument like
`go get golang.org/x/text` or we could provide patterns like
`go get golang.org/x/perf/cmd/...`

The argument is a pattern if it includes one or more `...` wildcards.
A pattern expands to all package directories found in the directory tree
with names matching the patterns.

So, `go get -u ./...` is saying "get all the dependencies that are found
in this directory (`./`) and recursively within that directory (`...`).